### DISCUSSION TITLE
word fryser

### DISCUSSION THREAD

**Discussion by emilia2402** [Eduap-com/WordMat#94] [Support] [answered]
hej wordmat jeg har et problem i forhold til når jeg vil regne ligner.
<img width="354" alt="image" src="https://user-images.githubusercontent.com/116557962/197504835-b83167ca-79f3-44f9-86a5-fff17d1cd184.png">
Den vil ikke lade mig bruge nogle af strukturer formlerne. Når man vælger en af formlerne så fryser Word.
Er der andre der har oplevet det?
Jeg har forsøgt at reinstaller Office, wordmat, nedgrader og opgrader det og det har ikke virket.

**Comment by Eduap-com** [comment, marked as answer]
Den menu du viser der er ikke en del af WordMat, men indbygget i Word, så det er formentlig et Word-problem
